ui/packages/catalog-ui-search/src/main/webapp/component/visualization/table/table-viz.view.js

```diff
@@ -55,14 +54,23 @@ function getFilenameFromContentDisposition(header) {
     return parts[1];
 }
 
+const adaptFetchData = async (response) => ({
+    data: await response.text(),
+    status: response.status,
+    getResponseHeader: (requestedHeader) => {
+        for(const header of response.headers.entries()) {
+            if (header === requestedHeader)
+                return header[1];
```

Comment:
✏️ `getFilenameFromContentDisposition` could be more generically renamed to `getHeaderValue` and then reused here

Reply:
Good point, though most of this is throw away adapted for jquery->fetch stuff